### ISSUE TITLE
Relationship of Taxon Properties and Darwin Core

### ISSUE THREAD

**Issue by AlasdairGray** [BioSchemas/specifications#348]
@frmichel can you clarify if `parentTaxon` and `taxonRank` were inspired by their equivalent terms in Darwin Core, i.e. `dwc:parentNameUsageID` and `dwc:taxonRank` respectively?

Also, was NCBI Taxon ontology considered in the design?

**Comment by frmichel** [comment]
Hi @AlasdairGray,

In early discussions (June 2018), we were considering creating a Taxon profile. In this context, [we discussed](https://docs.google.com/document/d/1fcN76j3-4Ry99Re-Z6TpBbgL9QO3On-ZqG1xWBOJLPM/) possible candidate properties to reuse from Darwin Core, TDWG, NCBI, Wikidata etc. At this point, I proposed [examples](https://github.com/BioSchemas/specifications/blob/master/Taxon/examples/Deprecated/) using ```dwc:taxonRank``` and ```tc:rank``` (TDWG), but ```ncbitaxon:has_rank``` could have been used too).

Later on, we decided to go for a Taxon type instead. While the Taxon profile could still be used to specify terms from third-party ontologies, we needed a minimal set of core properties to accompany the Taxon type. Hence we defined the taxonRank, parentTaxon and childTaxon properties.

Indeed, taxonRank and parentTaxon are clearly inspired by their equivalent, primarily in DwC but not only. The object of dwc:parentNameUsageID is a literal, as is the case for many other properties in DwC, whereas we want an object property.
NCBI was not relevant for parentTaxon as they model the child-parent relationship as a owl:subClassOf.

So far, there is one DwC property that is suggested in the Taxon profile: ```dwc:vernacularNames```, as we thought it was not needed as a "core property" in schema.org Taxon type.

Note also that taxonRank does not impose specific values for the ranks. Once could use those from TDWG, NCBI (e.g. http://purl.obolibrary.org/obo/NCBITaxon_species), Wikidata etc. (DwC does not propose URIs for ranks). See [example 0.4](https://github.com/BioSchemas/specifications/blob/master/Taxon/examples/Delphinapterus%20leucas_jsonld_0.4.json).

Hope that helps.

Franck.
